### PR TITLE
`is_private` and `is_global`

### PR DESCRIPTION
**Pull request by fredrikekre**
Add `is_private` and `is_global` for classification of IP addresses.

### CODE REVIEW

**Comment by fredrikekre** [comment]
(Sorry for the "mass review request". The repo requires approvals so I can not merge it without it :) )

**Review by quinnj — APPROVED** [review]
I also invited you to JuliaWeb admin team

**Comment by fredrikekre** [comment]
Thanks!

**Pull request merged**
